src/forms/subscriptions.ts

```diff
@@ -90,45 +103,66 @@ class FormFields extends BaseFormFields {
         this.addSubNameDependentFields();
     }
 
-    // setState sets state for teamTriggers and teamChannelsWithSubs
-    //   * teamTriggers - object with keys of channel IDs and values of
-    //                  array of triggers for the current team
-    //   * teamChannelsWithSubs - array of channels with subscriptions
+    // setState sets state for triggers and channelsWithSubs
     async setState(): Promise<void> {
+        const triggers = await this.getTriggers();
+        await this.buildChannelsWithSubs(triggers);
+        this.addChannelTrigger(triggers);
+    }
```

Comment:
It's not obvious to be that the second line here will mutate `triggers`, then we use the same variable on the third line. Maybe have the second line return `triggers` like:

```js
triggers = await this.buildChannelsWithSubs(triggers);
```